backend/src/tasks.ts

```diff
@@ -0,0 +1,349 @@
+        const receipt = await tx.wait();
+        console.log(`Success! Transaction hash: ${receipt!.hash}`);
+    });
+
+// Clear (delete) questions.
+task('clearQuestions')
+    .addPositionalParam('address', 'contract address')
+    .setAction(async (args, hre) => {
+        await hre.run('compile');
+
+        let quiz = await hre.ethers.getContractAt('Quiz', args.address);
+        const tx = await quiz.clearQuestions();
+        const receipt = await tx.wait();
+        console.log(`Success! Transaction hash: ${receipt!.hash}`);
+    });
+
+// Update existing question.
+task('setQuestion')
+    .addPositionalParam('address', 'contract address')
+    .addPositionalParam('number', 'question number (starting from 0)')
+    .addPositionalParam('questionsFile', 'file containing questions in JSON format')
+    .setAction(async (args, hre) => {
+        await hre.run('compile');
+
+        let quiz = await hre.ethers.getContractAt('Quiz', args.address);
+        const questions = JSON.parse(await promises.readFile(args.questionsFile, 'utf8'));
+        const tx = await quiz.setQuestion(args.number, questions[parseInt(args.number)].question, questions[parseInt(args.number)].choices);
+        const receipt = await tx.wait();
+        console.log(`Updated question ${questions[parseInt(args.number)].question}. Transaction hash: ${receipt!.hash}`);
+    });
+
+// Add questions from json file.
+task('addQuestions')
+    .addPositionalParam('address', 'contract address')
+    .addPositionalParam('questionsFile', 'file containing questions in JSON format')
+    .setAction(async (args, hre) => {
+        const quiz = await hre.ethers.getContractAt('Quiz', args.address);
+        await addQuestions(quiz, args.questionsFile);
+    });
+
+// Add coupons.
+task('addCoupons')
+    .addPositionalParam('address', 'contract address')
+    .addPositionalParam('couponsFile', 'file containing coupons, one per line')
+    .setAction(async (args, hre) => {
+        const quiz = await hre.ethers.getContractAt('Quiz', args.address);
+        await addCoupons(quiz, args.couponsFile);
+    });
+
+// Fetch image from IPFS.
+task('storeNFT')
+    .addPositionalParam('quiz', 'Quiz contract')
+    .addPositionalParam('jsonFile', 'Path to JSON metadata file')
+    .setAction(async (args, hre) => {
+        const quiz = await hre.ethers.getContractAt('Quiz', args.quiz);
+        const jsonMetadata = await promises.readFile(args.jsonFile, 'utf8');
+        await storeNFT(hre, quiz, jsonMetadata);
+    });
+
+// Add minter address, usually only allowed for Quiz contract.
+task('addAllowMint')
+    .addPositionalParam('rewardContract', 'Reward contract to update')
+    .addPositionalParam('minterAddress', 'Grant Mint access to this sender address')
+    .setAction(async (args, hre) => {
+        const oasisReward = await hre.ethers.getContractAt('OasisReward', args.rewardContract);
+        await addAllowMint(hre, oasisReward, args.minterAddress);
+    });
+
```

Comment:
For completeness, also add removeAllowMint.

Reply:
Added removeAllowMint.